src/engine/QueryPlanner.cpp

```diff
@@ -1583,13 +1589,122 @@ std::vector<SubtreePlan> QueryPlanner::runGreedyPlanningOnConnectedComponent(
   return result;
 }
 
+// _____________________________________________________________________________
+std::vector<SubtreePlan> QueryPlanner::seedImplicitSpatialJoin(
+    std::vector<SparqlFilter> filters) {
+  std::vector<SubtreePlan> plans;
+
+  // TODO<ullingerc> Add support for more optimizable filters:
+  // * geof:distance() < constant
+  // * constant > geof:distance()
+  // * constant >= geof:distance()
```

Comment:
Can't this TODO be handled in the `getGeoFunctions` call, and thus shouldn't be here?

Reply:
Ah I see, you do the check of the `less than` expressions here.